README.md

```diff
@@ -1,5 +1,10 @@
 # Cro [![Build Status](https://travis-ci.org/croservices/cro.svg?branch=master)](https://travis-ci.org/croservices/cro)
 
-This is part of the Cro libraries for implementing services and distributed
-systems in Perl 6. See the [Cro website](http://cro.services/) for further
-information and documentation.
+This repository contains the [`cro` command line tool], as well as the sources for the [Cro website](http://cro.services/) for further
```

Comment:
Technically, it is "sources for documentation on the website", not "sources for the website".

Reply:
Changed :-)